style.css

```diff
@@ -264,3 +265,132 @@ h2 {
   font-size: 40px;
   line-height: 52px;
 }
+
+#ContactSection {
+  background: #6070ff;
+  border-radius: 130px 0 0 0;
+  margin-left: 0;
+  margin-right: 0;
+  margin-top: 110px;
+  padding-top: 50px;
+  background-image: url("images/footerbg.png");
+  background-repeat: no-repeat;
+  background-position: 80px 10px;
+}
+
+#Contact-form {
+  display: flex;
+  flex-direction: column;
+  padding-left: 40px;
+  padding-right: 40px;
+  padding-bottom: 40px;
+}
+
+/* .inputclass{
+  align-items: center;
+  margin-bottom: 20px;
+  border: 1px solid #CFD8DC;
+  border-radius: 8px;
+  height:50px;
+  border:none;
+} */
```

Comment:
- [ ] [OPTIONAL] It is not a good practice to push redundant or commented code to remote. Kindly ensure you always remove all commented code before pushing.